src/components/notification/notification.jsx

```diff
@@ -0,0 +1,63 @@
+import styled from "styled-components";
+import { Avatar } from "../avatar/avatar";
+
+/**
+ * Notification Component
+ * @author [J. Hartsek](https://github.com/JHartsek)
+ * @param src the avatar image link
+ * @param alt the avatar image description
+ * @param title the notificaiton origin
+ * @param summary the notfication details
+ * @param status expects 'read' or 'unread'
+ */
+
+const NotificationWrapper = ({ children }) => ({ children });
+
+NotificationWrapper.Container = styled.div`
+  column-gap: 8px;
+  padding: 12px 0 16px 12px;
+`;
+
+NotificationWrapper.Info = styled.div`
+  row-gap: 4px;
+`;
+
+NotificationWrapper.Title = styled.h3`
+  margin: 0;
+  color: rgba(246, 246, 246, 0.7);
+  font-family: "DM Sans", sans-serif;
```

Comment:
Is this not getting applied by the default theme styles? It should be, but if not we should solve that problem instead of adding the font-family in places like this

Reply:
For some reason, it wasn't the other day, but it seems to be working now. I removed a couple other styles declarations that match the default as well.